### ISSUE TITLE
White screen in simlulator

### ISSUE THREAD

**Issue by Dharmoslap** [facebook/react-native#1214]
When I run the project in the simulator, I'm getting this white screen:

![screen shot 2015-05-09 at 14 55 31](https://cloud.githubusercontent.com/assets/3543614/7550275/7d0b1584-f65b-11e4-8c1a-59a21698455e.png)

Code in index.ios.js is the same as in the init example generated by react-native-cli.

And there is an error in console:

![screen shot 2015-05-09 at 16 40 47](https://cloud.githubusercontent.com/assets/3543614/7550650/3c2cb7d4-f66a-11e4-9959-3eb5ca55136c.png)

**Comment by kyledetella** [comment]
This is happening for me as well.

**Comment by droid-mind432** [comment]
i'm having a same issue

**Comment by GertjanReynaert** [comment]
@Dharmoslap @kyledetella @droid-mind432 What external packages are you using? I had this issue when I was using an external npm-module that wasn't updated to React Native 0.4.0. After updating that module the issue was resolved for me.

Hope this helps.

**Comment by Dharmoslap** [comment]
I was using  0.4.1, lately even 0.4.2 and it's the same. I'm running OS X 10.9.5 (13F1077) and xCode 6.2 (6C131e).

**Comment by GertjanReynaert** [comment]
@Dharmoslap I didn't meant your ReactNative version itself. I meant the ReactNative dependency of any of the npm modules you installed.

How I uncovered the issue: I created an npm module myself when RN was still in 0.3.x As I tried to update the RN version of my app (that uses my own npm module) I also got the white screen. When I updated my own npm modules RN dependency to 0.4.0 and reinstalled the module, the white screen was gone and the app was working fine. (I only found this out because I stumbled upon this comment: https://github.com/facebook/react-native/issues/917#issuecomment-94269441)

So my guess is that any of the modules in your `node_modules` folder still relies on RN 0.3.x
